public/data/Meyazhagan.json

```
@@ -0,0 +1,57 @@
+{
+    "name": "Meyazhagan",
+    "type": "personal",
+    "bio": "I'm a Full Stack Web Application Developer. I develop primarily with React and Node.js, C#, Python, TypeScript.",
+    "avatar": "https://github.com/Meyazhagan.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Meyazhagan",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/meyazhaganstwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think? :)
```suggestion
        "name": "Follow me on LinkedIn",
```